extensions/amp-selector/1.0/storybook/Basic.amp.js

```diff
@@ -72,20 +71,15 @@ export const WithUl = () => {
 
 WithUl.storyName = 'with <ul>';
```

Comment:
You can remove this. It's used to describe the story in more detail if needed, but in most cases in our storybook tests, it's the same as the test name.